librz/core/analysis_tp.c

```diff
@@ -274,11 +292,14 @@ RzAnalysisOp *op_cache_get(HtUP *cache, RzCore *core, ut64 addr) {
  */
 static void type_match(RzCore *core, char *fcn_name, ut64 addr, ut64 baddr, const char *cc,
 	int prev_idx, bool userfnc, ut64 caddr, HtUP *op_cache) {
-	Sdb *trace = core->analysis->esil->trace->db;
+	RzAnalysisEsilTrace *etrace = core->analysis->esil->trace;
 	RzTypeDB *typedb = core->analysis->typedb;
 	RzAnalysis *analysis = core->analysis;
 	RzList *types = NULL;
-	int idx = sdb_num_get(trace, "idx", 0);
+
+	// CHECK_ME : why use sdb to store idx before ?
```

Comment:
I think this comment is unnecessary now?

Reply:
Yes